### DISCUSSION TITLE
TShock configuring config.json

### DISCUSSION THREAD

**Discussion by Tr1gered** [Pryaxis/TShock#2100] [Q&A]
Russian:
Привет! Недавно связался с Tshock сервером и начал настраивать config.json (Конфиг сервера). При полной (настройке как я думал) запустил сервер играл и увидел что нашествия не призываются, питомцы и даже нельзя пересилить NPC (Мипов). Помогите как исправить 3 этих проблемы?
-Призыв нашествий.
-Призыв питомцев .
-Переселение NPC (Мипов).

English:
Hi! Recently contacted the Tshock server and started configuring config.json (server config). At full (setup as I thought) started the server played and saw that invasions are not called, Pets and even it is impossible to overpower NPC (MIPS). Help me fix these 3 problems?

Invocation of invasions.
Summon Pets.
Relocation of NPCs (MIPS).
This is translated from Russian to English there may be grammatical errors

**Comment by Quinci135** [comment]
You need to grant permissions to your default user group.
Your default user group is defined in config: `  "DefaultRegistrationGroupName": "default",`
You can add permissions to a permissions group with`/group addperm <group name> <list of permissions>`
In your case, `/group addperm default tshock.npc.spawnpets tshock.world.movenpc tshock.npc.startinvasion`
